lambda_backend/route_microservice/src/common/db.ts

```diff
@@ -0,0 +1,29 @@
+import DynamoDB, { AttributeValue, GetItemInput } from 'aws-sdk/clients/dynamodb';
+import { RouteItem } from './types';
+import createError from 'http-errors';
+
+const dynamoDb = new DynamoDB.DocumentClient();
+
+export const getItemFromRouteTable = async (
+  routeOwnerUsername: string,
+  createdAt: string,
+): Promise<RouteItem> => {
+  const getItemInput: GetItemInput = {
+    TableName: process.env['ROUTE_TABLE_NAME'] || '',
+    ConsistentRead: false,
+    Key: {
+      username: routeOwnerUsername as AttributeValue,
+      createdAt: createdAt as AttributeValue,
+    },
+  };
+  let Item;
+  try {
+    ({ Item } = await dynamoDb.get(getItemInput).promise());
```

Comment:
Are the brackets needed

Reply:
If you destructure while reassigning, yep